core/client/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -457,7 +457,11 @@ public void initialize(URI uri, org.apache.hadoop.conf.Configuration conf) throw
     try {
       statuses = sFileSystem.listStatus(uri);
     } catch (AlluxioException e) {
-      throw new IOException(e);
+      if (e instanceof FileDoesNotExistException) {
```

Comment:
Instead of doing `instanceof`, could we `catch (FileDoesNotExistException ...)` and then `catch (AlluxioException ...)`?